src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ActivityResourceIntTest.kt

```diff
@@ -100,7 +100,7 @@ class ActivityResourceIntTest : IntegrationTestBase() {
 
       @Test
       fun `invalid prison should return bad request`() {
-        val invalidPrison = validJsonRequest().replace(""""prisonId" : "LEI",""", """"prisonId" : "ZZX",""")
+        val invalidPrison = validJsonRequest().withPrison("ZZX")
```

Comment:
Lots of these to standardise JSON request creation already in use in some test classes - hide the details of JSON wrangling for tests to make the tests more readable.